wormhole-connect/src/views/v2/Bridge/AssetPicker/index.tsx

```diff
@@ -211,10 +214,12 @@ const AssetPicker = (props: Props) => {
             isFetching={props.isFetching}
             selectedChainConfig={chainConfig}
             selectedToken={props.token}
+            selectedTokenChain={selectedTokenChain}
             sourceToken={props.sourceToken}
             wallet={props.wallet}
             onSelectToken={(key: string) => {
               props.setToken(key);
+              setSelectedTokenChain(chainConfig?.key || '');
```

Comment:
would this work with an empty string as the argument? isn't it impossible to select a token with no chain selected?

Reply:
It is impossible. This was to prevent the TS error, but instead of setting it to empty string, I'll check `chainConfig?.key` and skip the call.

Reply:
Never mind, this block already has `!showChainSearch && chainConfig` above, so chainConfig will never be falsy.